packages/generator/src/generator.ts

```diff
@@ -267,7 +272,17 @@ export abstract class Generator<
     if (!this.options.destinationRoot) this.options.destinationRoot = process.cwd()
   }
 
-  abstract getTemplateValues(): Promise<any>
+  // abstract getTemplateValues(): Promise<any>
```

Comment:
please remove this commented line or add additional text to clarify why it is here.